frontend/src/components/projects/ProjectsDetailedContent.tsx

```diff
@@ -62,10 +54,14 @@ interface ProjectsDetailedContentProps {
 }
 
 const ProjectsDetailedContent = ({project, studies} : ProjectsDetailedContentProps) => {
+	const dispatch = useAppDispatch()
+
 	const navigate = useNavigate()
 
 	const [activeKey, setActiveKey] = useHashURL('overview')
 
+	const currentStudy = activeKey.startsWith("study") ? Number(activeKey.split("-")[1]) : null
```

Comment:
This seems very hacky to me and will break if someone decides to tweak the key string for some reason. I suggest creating a map of keys to study id's, and looking up the study ID that matches `activeKey`. That way, it won't break if the key format changes.